lms/templates/email/instructor_email_digest/body.html.jinja2

```diff
@@ -16,7 +16,8 @@
     <title>There has been some activity in Hypothesis</title>
     <style type="text/css">
  p{
-     margin:10px 0;
+     margin:0;
+     margin-bottom: 20px;
```

Comment:
Small tweak to the email CSS: put `20px` margin below `<p>`'s instead of `10px` margin above and `10px` below. This allows us to have an `<h3>` (the course title) before a `<p>` and to have no margin between the `<h3>` and the `<p>` (since `<h3>`'s have no `margin-bottom`. This is how I'm making course titles stand out a bit and look different from assignment titles, but having the course title still be spacially associated with the info about the course and its assignments. Otherwise it looks quite confusing if course titles don't look any different from assignment titles.

![image](https://github.com/hypothesis/lms/assets/22498/0fd82435-580c-485b-a7c9-1171724216ef)